parachain/runtime/interlay/src/lib.rs

```diff
@@ -1190,82 +1208,76 @@ construct_runtime! {
-        VaultCapacity: reward::<Instance3>::{Pallet, Storage, Event<T>} = 43,
-
-        Farming: farming::{Pallet, Call, Storage, Event<T>} = 44,
-        FarmingRewards: reward::<Instance4>::{Pallet, Storage, Event<T>} = 45,
-
-        // # Bitcoin SPV
-        BTCRelay: btc_relay::{Pallet, Call, Config<T>, Storage, Event<T>} = 50,
-        // Relay: 51
-
-        // # Operational
-        Security: security::{Pallet, Call, Config, Storage, Event<T>} = 60,
-        VaultRegistry: vault_registry::{Pallet, Call, Config<T>, Storage, Event<T>, ValidateUnsigned} = 61,
-        Oracle: oracle::{Pallet, Call, Config<T>, Storage, Event<T>} = 62,
-        Issue: issue::{Pallet, Call, Config<T>, Storage, Event<T>} = 63,
-        Redeem: redeem::{Pallet, Call, Config<T>, Storage, Event<T>} = 64,
-        Replace: replace::{Pallet, Call, Config<T>, Storage, Event<T>} = 65,
-        Fee: fee::{Pallet, Call, Config<T>, Storage} = 66,
-        // Refund: 67
-        Nomination: nomination::{Pallet, Call, Config, Storage, Event<T>} = 68,
-        ClientsInfo: clients_info::{Pallet, Call, Storage, Event<T>} = 69,
-
-        // # Governance
-        Democracy: democracy::{Pallet, Call, Storage, Config<T>, Event<T>} = 70,
-        TechnicalCommittee: pallet_collective::<Instance1>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>} = 71,
-        TechnicalMembership: pallet_membership::{Pallet, Call, Storage, Event<T>, Config<T>} = 72,
-        // Treasury: 73
-
-        Authorship: pallet_authorship::{Pallet, Storage} = 80,
-        CollatorSelection: collator_selection::{Pallet, Call, Storage, Event<T>, Config<T>} = 81,
-        Session: pallet_session::{Pallet, Call, Storage, Event, Config<T>} = 82,
-        Aura: pallet_aura::{Pallet, Storage, Config<T>} = 83,
-        AuraExt: cumulus_pallet_aura_ext::{Pallet, Storage, Config} = 84,
-        ParachainSystem: cumulus_pallet_parachain_system::{Pallet, Call, Config, Storage, Inherent, Event<T>} = 85,
-        ParachainInfo: parachain_info::{Pallet, Storage, Config} = 86,
-
-        // # XCM Helpers
-        XcmpQueue: cumulus_pallet_xcmp_queue::{Pallet, Call, Storage, Event<T>} = 90,
-        PolkadotXcm: pallet_xcm::{Pallet, Call, Storage, Event<T>, Origin, Config} = 91,
-        CumulusXcm: cumulus_pallet_xcm::{Pallet, Call, Event<T>, Origin} = 92,
-        DmpQueue: cumulus_pallet_dmp_queue::{Pallet, Call, Storage, Event<T>} = 93,
-
-        XTokens: orml_xtokens::{Pallet, Storage, Call, Event<T>} = 94,
-        UnknownTokens: orml_unknown_tokens::{Pallet, Storage, Event} = 95,
-
-        // # Lending & AMM
-        Loans: loans::{Pallet, Call, Storage, Event<T>, Config} = 100,
-        DexGeneral: dex_general::{Pallet, Call, Storage, Event<T>} = 101,
-        DexStable: dex_stable::{Pallet, Call, Storage, Event<T>}  = 102,
-        DexSwapRouter: dex_swap_router::{Pallet, Call, Event<T>} = 103,
+      System: frame_system::{Pallet, Call, Storage, Config, Event<T>} = 0,
```

Comment:
Seems the indentation changed here, please revert to avoid unnecessary code shift.